### PR TITLE
PDEP Decision Making

### PR DESCRIPTION
**Pull request by noatamir**
The governance working group has been discussing the PDEP workflow and decision-making for some time now.

This PR proposes some formulation of the existing workflow to clarify and facilitate it for current and future authors, and voting members.  
In our discussions, we assessed that a more structured process would help folks who submit the PDEP know how and when it can progress, and how to better engage with others.

The timelines were considered to facilitate an engaging discussion, and automation. We aim to make the lives of the PDEP authors easier by taking away the need to decide "has it been long enough", or "cat herding" folks to engage and clarify their comments so that PDEPs don't get large change requests that could have been clarified early on - late in the process.

The quorum was discussed a lot in our working group, and we're unsure about the best strategy here. We were looking for a sufficient number so if a decision is passed one feels it's valid (e.g. maybe 2 people is too low), but not too high so that we feel like ongoing discussions are blocked because it's too hard to get that many voting members involved. Further proposal on this point are welcome. 

We tried to make things as flexible and clear as possible. Being mindful of the needs of the authors, as well as those engaging in the discussion, be them voting members or not.

At this point, we would like to invite feedback. It might be that you need more clarity on the suggestions, have ideas for improvement, or aren't convinced that this will affect the current process positively.

### CODE REVIEW

File: web/pandas/pdeps/0001-purpose-and-guidelines.md [hunk 1]
```diff
@@ -71,17 +71,60 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed to
-`Status: Accepted` when the PDEP is ready and has the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. This will be changed to
```
Comment (jorisvandenbossche) [inline web/pandas/pdeps/0001-purpose-and-guidelines.md:85]: A little bit up, in the list of possible states, we should probably add "Draft"?

**Comment by jorisvandenbossche** [comment]
NEP and SLEP 0 both have the following paragraph:

> In general, the goal is to make sure that the community has consensus, not provide a rigid policy for people to try to game. When in doubt, err on the side of asking for more feedback and looking for opportunities to compromise.

I think I would prefer to include something like that as well here

File: web/pandas/pdeps/0001-purpose-and-guidelines.md [hunk 2]
```diff
@@ -71,17 +71,60 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
...
+`Status: Under discussion` by the author(s), when they are ready to proceed with the descision
+making process. 
+
```
Comment (jorisvandenbossche) [inline web/pandas/pdeps/0001-purpose-and-guidelines.md:90]: Could we have a subsection "How a PDEP becomes Accepted" (or different title), and then having schedule/voting/quorum be subsection of this one?

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
as discussed in governance meeting, add an introductory paragraph to the Workflow section that indicates the rationale for the process.

File: web/pandas/pdeps/0001-purpose-and-guidelines.md [hunk 3]
```diff
@@ -56,6 +56,12 @@ advisor on the PDEP when it is submitted to the PDEP repository.
 
 ### Workflow
 
+Our workflow was created to support and enable a consensus seeking process, and to provide clarify,
```
Comment (Dr-Irv) [inline web/pandas/pdeps/0001-purpose-and-guidelines.md:61]: make this subsection `#### Rationale`

File: web/pandas/pdeps/0001-purpose-and-guidelines.md [hunk 4]
```diff
@@ -56,6 +56,12 @@ advisor on the PDEP when it is submitted to the PDEP repository.
...
+for current and future authors, as well as voting members. It is not a strict policy, and we
+discourage any interpretation which seeks to take advantage of it in a way that could "force" or
+"sneak" decisions in one way or another. We expect and encourage transparency, active discussion,
+feedback, and compromise from all our community members.
+
 The possible states of a PDEP are:
```
Comment (Dr-Irv) [inline web/pandas/pdeps/0001-purpose-and-guidelines.md:69]: Make this a subsection `#### PDEP States`

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
small changes to create section headers

File: web/pandas/pdeps/0001-purpose-and-guidelines.md (hunk 2 above)
Comment (noatamir) [reply web/pandas/pdeps/0001-purpose-and-guidelines.md:90]: I find this a bit redundant as this is all under the PDEP workflow, and it just makes the following sections be on a different level which bugs me somehow(?!). I tried renaming a few of the subsections to increase clarify. Please let me know if you still think this is needed.

**Pull request closed without merging**